src/api/common.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(Serialize)]
+pub struct PaginatedResponse<T> {
+    pub limit: i32,
+    pub offset: i32,
+    pub data: T,
```

Comment:
isn't paginated data always in a list? if so, we should probably do `pub data: Vec<T>`. then we can say `PaginatedResponse<Game>` instead of `PaginatedResponse<Vec<Game>>`

Reply:
Yeah, that's fair. Will fix.